runtime/src/snapshot_package.rs

```diff
@@ -85,100 +80,12 @@ impl AccountsPackage {
             snapshot_links: snapshot_tmpdir,
             storages: snapshot_storages,
             hash: bank.get_accounts_hash(),
-            archive_format,
             snapshot_version,
-            snapshot_output_dir: snapshot_package_output_path.as_ref().to_path_buf(),
             expected_capitalization: bank.capitalization(),
             hash_for_testing,
             cluster_type: bank.cluster_type(),
         })
     }
-
-    /// Package up bank snapshot files, snapshot storages, and slot deltas for a full snapshot.
-    #[allow(clippy::too_many_arguments)]
-    pub fn new_for_full_snapshot(
```

Comment:
Can get rid of these two `new_for_full/incremental_snapshot()` functions. `AccountsPackage` no longer knows about full vs incremental snapshots, just `SnapshotPackage`.